src/app/shared/components/template/components/dashed-box/dashed-box.component.scss

```diff
@@ -11,11 +11,16 @@
     border: 1px dashed transparent;
     border-radius: var(--ion-border-radius-secondary);
     position: relative;
-    .text {
+    .text{
+      white-space: pre-line; 
       text-align: center;
       font-size: var(--font-size-text-medium);
       line-height: var(--line-height-text-largest);
-    }
+
+      p {
+        margin-bottom: 0.75em 0 ;
```

Comment:
I don't think this CSS rule is doing anything so it's not needed. As far as I can tell, the text that gets rendered is passed in as `innerHTML`, so there are no `<p>` elements inside `.text`. I would recommend deleting this rule unless there's a reason for it